selfdrive/modeld/modeld

```diff
@@ -1,10 +1,4 @@
 #!/usr/bin/env bash
 
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
-cd "$DIR/../../"
```

Comment:
this whole shim should be removed

Reply:
That's not possible yet because of:
https://github.com/tinygrad/tinygrad/blob/32c995a5daa7b20e5f01571d504a8b8807f866fe/tinygrad/device.py#L25

Reply:
I usually like to leave a comment in that case: "TODO: can be removed once <> is resolved"